src/internal/internalPane.ts

```diff
@@ -6,8 +6,11 @@
 /* global alert confirm */
 
 import { icons, ns, widgets } from 'solid-ui'
-import { IndexedFormula, literal, NamedNode, st, sym } from 'rdflib'
+// import { IndexedFormula, literal, NamedNode, st, sym } from 'rdflib'
+// import { IndexedFormula, literal, NamedNode } from 'rdflib' // import/no-duplicates
+import * as $rdf from 'rdflib'
 import { PaneDefinition } from 'pane-registry'
+const { IndexedFormula, NamedNode, literal } = $rdf
```

Comment:
What is internalPane, how is it related to the pad pane, and why do you propose we add the `$rdf` global to this file, rather than referring to the individual imports like `IndexedFormula`, `NamedNode`, etc directly, like it was?